var/spack/repos/builtin/packages/libxml2/package.py

```diff
@@ -40,7 +40,7 @@ class Libxml2(AutotoolsPackage):
 
     @property
     def headers(self):
-        include_dir = self.spec.prefix.include.libxml2
+        include_dir = self.spec.prefix.include
```

Comment:
I think this change will break other packages.

Reply:
The libxml2 dependents that I have on my system build fine with the change. I checked a handful of packages that I do not use and they also built fine. I have not, however, checked every libxml2 dependent. I do know that some of those packages are broken for other reasons. I can try to check them more systematically but that will take a while and I am not sure when I will be able to get to it.

In the meantime I will commit a change to this PR to back out the libxml2 header property change and comment out the libxml2 variant from ffmpeg. The libxml2 option in ffmpeg has limited scope so is likely not too important. I do not want that to hold up this PR.